cypress/TestCases/FireboltCertification/Accessibility2198.feature

```diff
@@ -0,0 +1,109 @@
+Feature: Accessibility
+
+    Background: Launch FCA for 'Accessibility'
+        Given the environment has been set up for 'Accessibility' tests
+        And 3rd party 'certification' app is launched
+
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario>
+        When '3rd party app' registers for the 'Firebolt' 'CLOSEDCAPTION_SETTINGS' event
```

Comment:
Let's use the establish GC created here: https://ccp.sys.comcast.net/browse/SYSTEST-9780 so we don't have to use `CLOSEDCAPTION_SETTINGS` in each test step

Reply:
updated with new test step